### ISSUE TITLE
Issue on parsing radial gradient in Compass 1.0

### ISSUE THREAD

**Issue by ai** [Compass/compass#1766]
When I try to parse by `sass --compass test.scss:test.css`:

``` css
a{
    background-image: radial-gradient(farthest-side at left top, #f4f4f4, rgba(244,244,244,0))
}
```

Sass throw:

```
home/ai/Dev/autoprefixer/benchmark/compass/.bundle/ruby/2.1.0/gems/sass-3.3.13/lib/sass/tree/visitors/cssize.rb:181:in `visit_prop': undefined method `empty?' for #<Sass::Script::Value::String:0x007f2f95dd8148> (NoMethodError)
    from /home/ai/Dev/autoprefixer/benchmark/compass/.bundle/ruby/2.1.0/gems/sass-3.3.13/lib/sass/tree/visitors/base.rb:36:in `visit'
    from /home/ai/Dev/autoprefixer/benchmark/compass/.bundle/ruby/2.1.0/gems/sass-3.3.13/lib/sass/tree/visitors/cssize.rb:23:in `visit'
    from /home/ai/Dev/autoprefixer/benchmark/compass/.bundle/ruby/2.1.0/gems/sass-3.3.13/lib/sass/tree/visitors/cssize.rb:43:in `block in visit_children_without_parent'
    from /home/ai/Dev/autoprefixer/benchmark/compass/.bundle/ruby/2.1.0/gems/sass-3.3.13/lib/sass/tree/visitors/cssize.rb:43:in `map'
    from /home/ai/Dev/autoprefixer/benchmark/compass/.bundle/ruby/2.1.0/gems/sass-3.3.13/lib/sass/tree/visitors/cssize.rb:43:in `visit_children_without_parent'
    from /home/ai/Dev/autoprefixer/benchmark/compass/.bundle/ruby/2.1.0/gems/sass-3.3.13/lib/sass/tree/visitors/cssize.rb:32:in `block in visit_children'
    from /home/ai/Dev/autoprefixer/benchmark/compass/.bundle/ruby/2.1.0/gems/sass-3.3.13/lib/sass/tree/visitors/cssize.rb:54:in `with_parent'
    from /home/ai/Dev/autoprefixer/benchmark/compass/.bundle/ruby/2.1.0/gems/sass-3.3.13/lib/sass/tree/visitors/cssize.rb:31:in `visit_children'
    from /home/ai/Dev/autoprefixer/benchmark/compass/.bundle/ruby/2.1.0/gems/sass-3.3.13/lib/sass/tree/visitors/base.rb:36:in `block in visit'
    from /home/ai/Dev/autoprefixer/benchmark/compass/.bundle/ruby/2.1.0/gems/sass-3.3.13/lib/sass/tree/visitors/cssize.rb:217:in `visit_rule'
    from /home/ai/Dev/autoprefixer/benchmark/compass/.bundle/ruby/2.1.0/gems/sass-3.3.13/lib/sass/tree/visitors/base.rb:36:in `visit'
    from /home/ai/Dev/autoprefixer/benchmark/compass/.bundle/ruby/2.1.0/gems/sass-3.3.13/lib/sass/tree/visitors/cssize.rb:23:in `visit'
    from /home/ai/Dev/autoprefixer/benchmark/compass/.bundle/ruby/2.1.0/gems/sass-3.3.13/lib/sass/tree/visitors/cssize.rb:43:in `block in visit_children_without_parent'
    from /home/ai/Dev/autoprefixer/benchmark/compass/.bundle/ruby/2.1.0/gems/sass-3.3.13/lib/sass/tree/visitors/cssize.rb:43:in `map'
    from /home/ai/Dev/autoprefixer/benchmark/compass/.bundle/ruby/2.1.0/gems/sass-3.3.13/lib/sass/tree/visitors/cssize.rb:43:in `visit_children_without_parent'
    from /home/ai/Dev/autoprefixer/benchmark/compass/.bundle/ruby/2.1.0/gems/sass-3.3.13/lib/sass/tree/visitors/cssize.rb:32:in `block in visit_children'
    from /home/ai/Dev/autoprefixer/benchmark/compass/.bundle/ruby/2.1.0/gems/sass-3.3.13/lib/sass/tree/visitors/cssize.rb:54:in `with_parent'
    from /home/ai/Dev/autoprefixer/benchmark/compass/.bundle/ruby/2.1.0/gems/sass-3.3.13/lib/sass/tree/visitors/cssize.rb:31:in `visit_children'
    from /home/ai/Dev/autoprefixer/benchmark/compass/.bundle/ruby/2.1.0/gems/sass-3.3.13/lib/sass/tree/visitors/base.rb:36:in `block in visit'
    from /home/ai/Dev/autoprefixer/benchmark/compass/.bundle/ruby/2.1.0/gems/sass-3.3.13/lib/sass/tree/visitors/cssize.rb:65:in `visit_root'
    from /home/ai/Dev/autoprefixer/benchmark/compass/.bundle/ruby/2.1.0/gems/sass-3.3.13/lib/sass/tree/visitors/base.rb:36:in `visit'
    from /home/ai/Dev/autoprefixer/benchmark/compass/.bundle/ruby/2.1.0/gems/sass-3.3.13/lib/sass/tree/visitors/cssize.rb:23:in `visit'
    from /home/ai/Dev/autoprefixer/benchmark/compass/.bundle/ruby/2.1.0/gems/sass-3.3.13/lib/sass/tree/visitors/base.rb:24:in `visit'
    from /home/ai/Dev/autoprefixer/benchmark/compass/.bundle/ruby/2.1.0/gems/sass-3.3.13/lib/sass/tree/visitors/cssize.rb:6:in `visit'
    from /home/ai/Dev/autoprefixer/benchmark/compass/.bundle/ruby/2.1.0/gems/sass-3.3.13/lib/sass/tree/root_node.rb:38:in `css_tree'
    from /home/ai/Dev/autoprefixer/benchmark/compass/.bundle/ruby/2.1.0/gems/sass-3.3.13/lib/sass/tree/root_node.rb:20:in `render'
    from /home/ai/Dev/autoprefixer/benchmark/compass/.bundle/ruby/2.1.0/gems/sass-3.3.13/lib/sass/engine.rb:274:in `render'
    from /home/ai/Dev/autoprefixer/benchmark/compass/.bundle/ruby/2.1.0/gems/sass-3.3.13/lib/sass/exec.rb:384:in `process_result'
    from /home/ai/Dev/autoprefixer/benchmark/compass/.bundle/ruby/2.1.0/gems/sass-3.3.13/lib/sass/exec.rb:43:in `parse'
    from /home/ai/Dev/autoprefixer/benchmark/compass/.bundle/ruby/2.1.0/gems/sass-3.3.13/lib/sass/exec.rb:22:in `parse!'
    from /home/ai/Dev/autoprefixer/benchmark/compass/.bundle/ruby/2.1.0/gems/sass-3.3.13/bin/sass:13:in `<top (required)>'
    from /home/ai/Dev/autoprefixer/benchmark/compass/.bundle/ruby/2.1.0/bin/sass:23:in `load'
    from /home/ai/Dev/autoprefixer/benchmark/compass/.bundle/ruby/2.1.0/bin/sass:23:in `<main>'
```

It is Compass issue, because, when I parse by `sass test.scss:test.css` it will be OK.

**Comment by ai** [comment]
@chriseppstein @scottdavis this is part of GitHub CSS (I benchmark Autoprefixer, Compass and nib with it). Old Compass 0.12 parsed this part.

**Comment by ai** [comment]
Even

``` css
a {
    background-image: radial-gradient(white, black)
}
```

throw error. Maybe I forget some option?

BTW, my `Gemfile.lock` for Ruby 2.1:

```
GEM
  remote: https://rubygems.org/
  specs:
    chunky_png (1.3.1)
    compass (1.0.0)
      chunky_png (~> 1.2)
      compass-core (~> 1.0.0)
      compass-import-once (~> 1.0.5)
      rb-fsevent (>= 0.9.3)
      rb-inotify (>= 0.9)
      sass (>= 3.3.13, < 3.5)
    compass-core (1.0.0)
      multi_json (~> 1.0)
      sass (>= 3.3.0, < 3.5)
    compass-import-once (1.0.5)
      sass (>= 3.2, < 3.5)
    ffi (1.9.3)
    multi_json (1.10.1)
    rb-fsevent (0.9.4)
    rb-inotify (0.9.5)
      ffi (>= 0.5.0)
    sass (3.3.14)

PLATFORMS
  ruby

DEPENDENCIES
  compass
```

**Comment by nDmitry** [comment]
Have this issue too.

**Comment by mullerzs** [comment]
Try using @include background(radial-gradient(...)), it did the trick for me.

**Comment by deansimcox** [comment]
I had this problem too, like @mullerzs said, using the compass @include seems to let compass pass.

I'd love to know what the actual problem was though, I just wasted most of the day..

**Comment by chriseppstein** [comment]
fixed.

**Comment by chriseppstein** [comment]
Just released 1.0.1 with this fix.

**Comment by deansimcox** [comment]
Cheers @chriseppstein
